code/game/objects/items/stacks/tape.dm

```diff
@@ -82,6 +83,34 @@
 
 	return ITEM_INTERACT_SUCCESS
 
+/obj/item/stack/sticky_tape/interact_with_atom_secondary(atom/interacting_with, mob/living/user, list/modifiers)
+	if(!object_repair_value)
+		return NONE
```

Comment:
Any reason for this to be on the base tape class and not on the duct tape subclass, if it only works there anyways?

Reply:
Part of the reason is that I might come back and add an additional tape here using this functionality down the line. Another reason is that I feel like maybe sticky tape _should_ have this functionality as well but be really poor at repairing stuff compared to duct tape. I wasn't super sure about it and was gonna bounce the idea around.

Reply:
ah bugger it I'll move it to duct tape for now.